test/functional/multifactor_auths_controller_test.rb

```diff
@@ -239,45 +596,45 @@ class MultifactorAuthsControllerTest < ActionController::TestCase
                            verify_session_path]
       end
 
-      context "user has mfa set to weak level" do
-        setup do
-          @seed = ROTP::Base32.random_base32
-          @user.enable_totp!(@seed, :ui_only)
-        end
-
-        should "redirect user back to mfa_redirect_uri after successful mfa setup" do
-          @redirect_paths.each do |path|
-            session[:mfa_redirect_uri] = path
-            post :update, params: { otp: ROTP::TOTP.new(@seed).now, level: "ui_and_api" }
-
-            assert_redirected_to path
-            assert_nil session[:mfa_redirect_uri]
-          end
-        end
-
-        should "not redirect user back to mfa_redirect_uri after failed mfa setup, but mfa_redirect_uri unchanged" do
-          @redirect_paths.each do |path|
-            session[:mfa_redirect_uri] = path
-            post :update, params: { otp: "12345", level: "ui_and_api" }
-
-            assert_redirected_to edit_settings_path
-            assert_equal path, session[:mfa_redirect_uri]
-          end
-        end
-
-        should "redirect user back to mfa_redirect_uri after a failed setup + successful setup" do
-          @redirect_paths.each do |path|
-            session[:mfa_redirect_uri] = path
-            post :update, params: { otp: "12345", level: "ui_and_api" }
-
-            assert_redirected_to edit_settings_path
-            post :update, params: { otp: ROTP::TOTP.new(@seed).now, level: "ui_and_api" }
-
-            assert_redirected_to path
-            assert_nil session[:mfa_redirect_uri]
-          end
-        end
-      end
+      # context "user has mfa set to weak level" do
```

Comment:
These tests were failing because there's an extra mfa step when updating the mfa level. These can be removed as no one should be on `ui_only` anymore.

Reply:
We decided to keep these tests as these tests should be kept until the cleanup of `ui_only` is done.